### PR TITLE
*: remove unused profile memory tracker from global tracker

### PR DESCRIPTION
**Pull request by Yisaer**
Signed-off-by: yisaer <disxiaofei@163.com>

### What problem does this PR solve?

ref https://github.com/pingcap/tidb/issues/29264

The `kvcache` memory usage detected in profile memory tracker isn't accurate, thus we decided to remove it from global tracker.

### Release note

```release-note
Deprecated `performance.mem-profile-interval` as it's no longer used.
```

### CODE REVIEW

**Review by wshwsh12 — APPROVED** [review]
LGTM

File: util/profile/trackerRecorder.go [hunk 1]
```diff
@@ -25,21 +25,21 @@ import (
 var col = &Collector{}
 
 // HeapProfileForGlobalMemTracker record heap profile data into each global function memory tracker
-func HeapProfileForGlobalMemTracker(d time.Duration) {
+func HeapProfileForGlobalMemTracker(profileName string, d time.Duration) {
```
Comment (nolouch) [inline util/profile/trackerRecorder.go:28]: Can this function be deleted?

Comment (Yisaer) [reply util/profile/trackerRecorder.go]: updated.

**Review by XuHuaiyu — APPROVED** [review]
LGTM

**Pull request merged**